<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectWizard</class>
 <widget class="QWizard" name="ConnectWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Setup</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="options">
   <set>QWizard::HaveFinishButtonOnEarlyPages|QWizard::HelpButtonOnRight|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="connectionPage">
   <property name="title">
    <string>Connection</string>
   </property>
   <property name="subTitle">
    <string>Enter the connection details.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP-Address</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="IPAddressLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="portLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Protocol</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="tcpButton">
         <property name="text">
          <string>TCP</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="udpButton">
         <property name="text">
          <string>UDP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="selectionPage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Coming Soon: Signal-/Channel-Selection</string>
   </property>
   <property name="subTitle">
    <string>Select the signals and channels you want to see.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="signalTree">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="filterPage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Coming Soon: Filters</string>
   </property>
   <property name="subTitle">
    <string>Select filters.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
